index.html

```diff
@@ -0,0 +1,146 @@
+                    <p class="first-para">Lorem ipsum dolor sit amet consectetur adipisicing elit.
+                        Distinctio expedita nesciunt sunt facere aperiam quasi consequuntur culpa
+                        dolorem ab! Placeat, dicta laudantium!
+                    </p>
+                   
+                </div>
+            <div class="dates">
+                        <h2 class="dat-1">2022.10.15(THUR)~16(FRI)</h2>
+                        <P class="dat-para">@Mulungushi Internatinal Conference Center, Lusaka.</P>
+                    </div>
+            </div>
+
+        </section>
+    </section>
+    <section class="program-main-sec">
+        <div class="main-pro">
+            <h3 class="pro-heading">Main Program</h3>
+            <hr class="main-hr">
+        </div>
+        <div class="cards-desk">
+        <div class="programs">
+            <img class="program-img1" src="assets/images/program1.png" alt="program icon">
+            <h4 class="program-head">lecture</h4>
+            <p class="program-para">Lorem ipsum dolor sit amet consectetur adipisicing
+                elit. Minus architecto </p>
+        </div>
+        <div class="programs">
+            <img class="program-img1" src="assets/images/program2.png" alt="program icon">
+            <h4 class="program-head">lecture</h4>
+            <p class="program-para">Lorem ipsum dolor sit amet consectetur adipisicing
+                elit. Minus architecto </p>
+        </div>
+        <div class="programs">
+            <img class="program-img1" src="assets/images/program3.png" alt="program icon">
+            <h4 class="program-head">lecture</h4>
+            <p class="program-para">Lorem ipsum dolor sit amet consectetur adipisicing
+                elit. Minus architecto </p>
+        </div>
+        <div class="programs">
+            <img class="program-img1" src="assets/images/program4.png" alt="program icon">
+            <h4 class="program-head">lecture</h4>
+            <p class="program-para">Lorem ipsum dolor sit amet consectetur adipisicing
+                elit. Minus architecto </p>
+        </div>
+        <div class="programs">
+            <img class="program-img1" src="assets/images/program5.png" alt="program icon">
+            <h4 class="program-head">lecture</h4>
+            <p class="program-para">Lorem ipsum dolor sit amet consectetur adipisicing
+                elit. Minus architecto </p>
+        </div>
+        <div class="reseve-sport">
+           
+        </div>
+      </div>
+      <div class="btn-10">
+       <button class="reserve-btn">Reseve Your Sport</button>
+      </div>
+       <div class="desk-btn">
+       <a class="reserve-btn1" href="#">SEE THE WHOLE PROGRAM</a> 
+        </div>
+    </section>
+<div class="main-speakers">
+            <h3 class="spk-heading">Featured Speakers</h3>
+            <hr class="spk-hr">
+            </div> 
+    <section class="spakers-section" id="speakers">
+        
+    </section>
```

Comment:
- [ ] In the speakers' section, each image has different width and height by default. So, in order to make all the images the same, please add custom widths and heights to align them.

Reply:
- [ ] [REQUIRED] Great job fixing the other recommended changes, however, this needs to be fixed as recommended by the previous reviewer. The speaker images still do not look uniform. Kindly ensure this s fixed.
### Yout implementation
![Screenshot from 2022-04-20 17-36-10](https://user-images.githubusercontent.com/58692590/164281054-4a1d87fc-1562-4613-8c86-e76c7bf8c2fc.png)